wordpress-mobile/gutenberg-mobile
# [iOS] Fix empty rich-text with no height on RTL layout

Fixes #

PR to test https://github.com/WordPress/gutenberg/pull/24510
WPiOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/14630

![RTL](https://user-images.githubusercontent.com/9772967/89987364-403dbf80-dc7e-11ea-9e12-c0efa5f9331c.png)

### To test:
- Run the project from https://github.com/wordpress-mobile/WordPress-iOS/pull/14630.
- Set the Application Language to a RTL language (Hebrew - Arabic) on the Run Options on Xcode.
- Open a new empty post.
  - Check that the Title and inserter placeholders are visible.
  - Check that the fields are selectable and you can type on them.
  - Check that longer text makes the text area increase in height as expected.
- Make the same checks from ⬆️ on a LTR language.

PR submission checklist:
